jvm/docker-entrypoint.sh

```diff
@@ -3,7 +3,10 @@
 BENCHMARK="${1:-dummy}"
 # Use a fixed heap size
 # Defaults for heap from https://github.com/renaissance-benchmarks/measurements#measurement-information
-JAVA_OPTS="-Xms${HEAP_SIZE:-12288}m -Xmx${HEAP_SIZE:-12288}m -XX:+UseG1GC"
+JAVA_OPTS="-Xms${HEAP_SIZE:-12288}m -Xmx${HEAP_SIZE:-12288}m"
+if [ -n "$GC_COLLECTOR" ]; then
+	JAVA_OPTS="$JAVA_OPTS -XX:+Use${GC_COLLECTOR:-G1}GC"
+fi
```

Comment:
You don't need the `if` block because you have a default for `GC_COLLECTOR`.

Reply:
Very true. Fixing.